modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/view_import.jsp

```diff
@@ -16,27 +16,32 @@
 
 <%@ include file="/init.jsp" %>
 
-<%
-String redirect = ParamUtil.getString(request, "redirect");
-
-if (Validator.isNull(redirect)) {
-	redirect = PortletURLBuilder.createRenderURL(
-		renderResponse
-	).setMVCRenderCommandName(
-		"/sxp_blueprint_admin/view_sxp_blueprints"
-	).buildString();
-}
-%>
+<liferay-ui:icon
+	id="importIcon"
+	message="import"
+	onClick='<%= liferayPortletResponse.getNamespace() + "openImportModal();" %>'
+	url="javascript:void(0);"
+/>
 
 <div>
-	<span aria-hidden="true" class="loading-animation"></span>
-
 	<react:component
 		module="sxp_blueprint_admin/js/view_sxp_blueprints/ImportSXPBlueprintModal"
 		props='<%=
 			HashMapBuilder.<String, Object>put(
-				"redirectURL", redirect
+				"componentId", liferayPortletResponse.getNamespace() + "importModal"
```

Comment:
Thanks for passing this as a prop! 👍

Reply:
Np! Thanks for catching the file organization, added it in 💯